tools/civmanual.cpp

```diff
@@ -644,6 +644,7 @@ static bool manual_command(struct tag_types *tag_info)
       break;
 
     case MANUAL_COUNT:
```

Comment:
Why do we need to keep this MANUAL_COUNT again ? Some backward compatibility ? If yes, we need to comment it.

Reply:
It's the upper bound for the loop iterating over all manual types.